### DISCUSSION TITLE
How to add Module-Federation to existing Angular app?

### DISCUSSION THREAD

**Discussion by MaurizioCasciano** [nrwl/nx#23376] [Questions]
Here https://nx.dev/recipes/angular/dynamic-module-federation-with-angular#creating-a-new-dynamic-host-application
it is described how to create new Angular apps with Module-Federation support (both host and remote).

Is it possible to add Module-Federation to an existing Angular app ?

**Comment by leosvelperez** [comment]
Yes. You can use the `@nx/angular:setup-mf` generator. See the docs for the generator: https://nx.dev/nx-api/angular/generators/setup-mf.
